src/Components/Navigation.js

```diff
@@ -21,30 +17,37 @@ const Navigation = ({
   const idRight = dataRight === undefined ? 1 : dataRight.number;
 
   return (
-    <StyledDV>
-      <StyledButtons type="button" onClick={() => onNavigate(idLeft)}>
+    <StyledNavigation>
+      <StyledNavigationButtons type="button" onClick={() => onNavigate(idLeft)}>
         <Icon icon="ant-design:arrow-left-outlined" width="14" />
         {idLeft} · {textLeft}
-      </StyledButtons>
-      <StyledButtons type="button" onClick={() => onNavigate(idRight)}>
+      </StyledNavigationButtons>
+      <StyledNavigationButtons
+        type="button"
+        onClick={() => onNavigate(idRight)}
+      >
         {textRight} · {idRight}
         <Icon icon="ant-design:arrow-right-outlined" width="14" />
-      </StyledButtons>
-    </StyledDV>
+      </StyledNavigationButtons>
+    </StyledNavigation>
   );
 };
 
-const StyledButtons = styled.button`
-  font-size: 14px;
+const StyledNavigationButtons = styled.button`
+  font-size: 15px;
   color: whitesmoke;
   background: #1c1f26;
-  padding: 10px 10px;
+  padding: 12px;
   display: flex;
   justify-content: center;
   width: 100%;
+  @media (min-width: 667px) {
+    font-size: 17px;
+    padding: 13px;
+  }
 `;
 
-const StyledDV = styled.div`
+const StyledNavigation = styled.div`
```

Comment:
If it's a navigation (and links to other parts of the app), I would assume a `nav` element here, but it's rather a wrapper for filter button, right?